uvloop/handles/udp.pyx

```diff
@@ -244,14 +244,20 @@ cdef class UDPTransport(UVBaseTransport):
                 ctx.close()
 
                 exc = convert_error(err)
-                self._fatal_error(exc, True)
+                if isinstance(exc, OSError):
+                    self._protocol.error_received(exc)
```

Comment:
This needs to be wrapped with `run_in_context...()`, see also the `_on_sent()` below.